specification/app/resource-manager/Microsoft.App/stable/2025-07-01/examples/ContainerApps_Kind_CreateOrUpdate.json

```diff
@@ -0,0 +1,200 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "rg",
+    "containerAppName": "testcontainerAppKind",
+    "api-version": "2025-07-01",
+    "containerAppEnvelope": {
+      "location": "East Us",
+      "managedBy": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.Web/sites/testcontainerAppKind",
+      "kind": "workflowapp",
+      "properties": {
+        "managedEnvironmentId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/testmanagedenv3",
+        "configuration": {
+          "activeRevisionsMode": "Single",
+          "ingress": {
+            "external": true,
+            "targetPort": 80,
```

Comment:
>     "targetPort": 80, [](http://example.com/codeflow?start=7&length=22)

consider making your example show a secure HTTPS configuration instead of HTTP. Or at least having two examples, one secure, one insecure.